.changeset/odd-buttons-pay.md

```diff
@@ -2,14 +2,21 @@
 "astro": minor
 ---
 
-Adds support for passing a metastring when using the Code component.
+Adds a new property `meta` to Astro's built-in `<Code />` component.
 
-The following code is equivalent to doing `` ```js astro=cool `` in Markdown.
+This allows you to provide a value for [Shiki's `meta` attribute](https://shiki.style/guide/transformers#meta) to pass options to transformers.
+
+The following example passes an option to highlight lines 1 and 3 to Shiki's `tranformerMetaHighlight`:
 
 ```astro
+// src/pages/index.astro
```

Comment:
This probably would not be a page, since it doesn't have a layout and just calls a component, no?

Reply:
This is invalid Astro markup. Comments above the `---` mark are not allowed.

Reply:
Sure! We could fix this by changing it to be a component URL, or by adding some HTML comment for "missing stuff". I'll update to a component name, but the other would be fine if you want to suggest it!